### PR TITLE
Add introduction about Audit Log

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### What is changed, added or deleted? (Required)

添加关于审计日志的描述

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: new-audit-log.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
+---
+title: TiDB 审计日志
+summary: 了解使用 TiDB 的审计日志功能。
+aliases: ['/docs-cn/dev/audit-log/']
+---
+
+# TiDB 审计日志
+
+审计日志功能是 TiDB 的企业版特性，可以灵活地记录对 TiDB 服务器的各种操作，方便安全、运维人员查看 SQL 操作日志，及时发现问题。
```
Comment (songrijie) [inline new-audit-log.md:9]:
```suggestion
数据库审计是 TiDB 企业版的重要功能，为企业提供了一个强大的监管和审计工具。审计可以记录所有的查询语句、事务操作、用户登录等信息，并且可以将这些信息存储到审计日志中，以备将来的查询和分析，从而保证企业的数据安全和合规性。数据库审计功能可以帮助企业管理人员追踪数据库操作的来源和影响，确保数据不被非法窃取或篡改。同时，数据库审计还可以帮助企业遵守各种法规和合规要求，保障企业在法律和道德方面的合规性。
```

File: new-audit-log.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+
+> **注意：**
+>
+> TiDB 社区版用户无法使用审计日志功能，请[选择 TiDB 企业版](https://cn.pingcap.com/product/#SelectProduct)以使用该功能并享受商业专家的支持服务。
```
Comment (songrijie) [inline new-audit-log.md:13]:
```suggestion
> 数据库审计是 [TiDB 企业版](https://cn.pingcap.com/product/#SelectProduct) 提供的功能，请[咨询 TiDB 顾问团队](https://cn.pingcap.com/product/#SelectProduct)以获取 TiDB 企业版，同时享受商业专家的支持服务。
```

File: new-audit-log.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+* `SET GLOBAL tidb_audit_enabled = 1` 对应的事件类型是 `AUDIT`、`AUDIT_SET_SYS_VAR` 和 `AUDIT_ENABLE`。`AUDIT_ENABLE` 属于 `AUDIT_SET_SYS_VAR` 的子类型，`AUDIT_SET_SYS_VAR` 属于 `AUDIT` 的子类型
+
+使用事件类型来区别不同的 SQL 操作有以下的优点：
+
+* 不同的事件类型的审计日志记录[不同的信息](#日志记录信息)，比如 `CONNETION` 事件需要记录客户端的 IP 地址、端口等信息，而 `SELECT` 事件则需要记录查询的 SQL 语句、访问的表等信息。
+* 可以根据不同的事件类型来选择 TiDB 用户关心的 SQL 语句，[过滤掉不需要的审计日志](#日志过滤与规则)。
+
+TiDB 审计日志有以下事件类型：
+
+| **事件类型** | **描述** |
```
Comment (songrijie) [inline new-audit-log.md:30]: 这个表是不是加一列"父类型”会更加清楚一些？

Comment (CbcWestwolf) [reply new-audit-log.md]: 已添加。也添加了系统变量、函数和动态权限的描述，PTAL

File: new-audit-log.md [hunk 4]
```diff
@@ -0,0 +1,537 @@
...
+### 通用信息
+
+所有类型的审计日志均包含以下日志信息：
+
+| **信息** | **描述** |
+|---|---|
+| ID | 标识该操作审计记录的唯一标识符 |
+| EVENT | 该操作审计记录对应的事件类型，多个事件类型之间以逗号（`,`）分隔 |
+| USER | 该操作审计记录对应的用户名 |
+| ROLES | 上述用户所拥有的角色列表 |
```
Comment (songrijie) [inline new-audit-log.md:63]:
```suggestion
| ROLES | 用户在执行操作时所拥有的角色列表 |
```

File: new-audit-log.md [hunk 5]
```diff
@@ -0,0 +1,537 @@
...
+当事件类型为 `AUDIT` 或其子类型时，审计日志将记录以下信息：
+
+| **信息** | **描述** |
+|---|---|
+| AUDIT_OP_TARGET | TiDB 审计日志相关设置的对象，包括系统变量（比如 `tidb_audit_enabled`）和函数（比如 `audit_log_create_filter`） |
+| AUDIT_OP_ARGS | TiDB 审计日志相关设置的参数，比如开启审计日志功能时设置 `tidb_audit_enabled` 为 `ON`；或者通过 `audit_log_remove_filter` 删除过滤器时，参数为删除的过滤器的名字 |
+
+## 审计日志过滤与规则
+
+可以通过过滤器（filter）与过滤规则（rule）对记录到 TiDB 审计日志中的记录进行选择。
```
Comment (songrijie) [inline new-audit-log.md:107]:
```suggestion
用户通过过滤器（filter）与过滤规则（rule）对记录到 TiDB 审计日志中的记录进行设置。 "过滤器"定义过滤规则，包括需要记录的事件、目标数据库对象等； "过滤规则" 创建"过滤器"所作用的数据库用户。 多个数据库用户可以共享一个"过滤器"，用这种方式能够实现定义的共享，比如企业可能存在几种安全级别的用户，我们可以为每种安全级别创建"过滤器"，同一安全级别的用户能够共享同一个"过滤器"。 当安全规范更新时，只需要更新相应的"过滤器"即可。
```

File: new-audit-log.md [hunk 6]
```diff
@@ -0,0 +1,537 @@
...
+
+### 过滤器
+
+过滤器可以从以下几个方面对审计日志进行过滤。
+
+* 审计事件类型：包括包含的事件类型（`class`）与排除的事件类型（`class_excl`）。如果 `class` 与 `class_excl` 均未声明，则所有类型的时间将被记录在审计日志中
```
Comment (songrijie) [inline new-audit-log.md:113]:
```suggestion
* 审计事件类型：包括包含的事件类型（`class`）与排除的事件类型（`class_excl`）。如果 `class` 与 `class_excl` 均未声明，则所有类型的事件将被记录在审计日志中
```

Comment (songrijie) [inline new-audit-log.md:113]:
```suggestion
* 审计事件类型：包括包含的事件类型（`class`）与排除的事件类型（`class_excl`）。如果 `class` 与 `class_excl` 均未声明，则所有类型的事件将被记录在审计日志中
```

File: new-audit-log.md [hunk 7]
```diff
@@ -0,0 +1,537 @@
...
+>
+> `AUDIT` 事件类型的审计日志会强制记录，无法通过过滤器排除。
+
+### 过滤规则
+
+创建过滤器之后，可以通过过滤规则将过滤器与用户进行绑定。绑定后的用户的操作会经过该过滤器决定是否记录到审计日志中，也可以方便地禁用/启用该过滤器。
+
+一条过滤规则由审计用户与过滤器名组成。
+
+* 审计用户由用户名与用户主机名组成，并以 `@` 作为用户名与用户主机名的分隔符。
```
Comment (songrijie) [inline new-audit-log.md:179]:
```suggestion
* 审计用户由用户名与用户主机名组成，并以 `@` 作为用户名与用户主机名的分隔符。和数据库用户定义相同，主机名部分可以使用`%`和`_`作为通配符，匹配到最长用户名的规则生效。 
```

File: new-audit-log.md [hunk 8]
```diff
@@ -0,0 +1,537 @@
...
+
+`JSON` 格式的审计日志示例如下：
+
+```json
+{"TIME":"2023/03/13 16:51:30.660 +08:00","ID":"851e8de3-b016-4384-8440-7386c033ef54-002f","EVENT":["QUERY","SELECT"],"USER":"root","ROLES":[],"CONNECTION_ID":"2199023255955","TABLES":["`test`.`t`"],"STATUS_CODE":1,"CURRENT_DB":"test","SQL_TEXT":"select * from `t`"}
+```
+
+## 日志轮替
+
+[日志轮替（log rotation）](https://zh.wikipedia.org/wiki/%E6%97%A5%E5%BF%97%E8%BD%AE%E6%9B%BF) 可以用于限制日志文件的大小。TiDB 中可以使用 [`tidb_audit_log_max_size`](#tidb_audit_log_max_size) 设置单个审计日志文件的大小，可以通过 [`tidb_audit_log_max_lifetime`](#tidb_audit_log_max_lifetime) 设置触发审计日志轮替的时间，也可以通过 [`audit_log_rotate`](#audit_log_rotate) 函数手动触发一次日志轮替。
```
Comment (songrijie) [inline new-audit-log.md:210]:
```suggestion
[日志轮替（log rotation）](https://zh.wikipedia.org/wiki/%E6%97%A5%E5%BF%97%E8%BD%AE%E6%9B%BF) 可以用于限制日志文件的大小。TiDB 中可以使用 [`tidb_audit_log_max_size`](#tidb_audit_log_max_size) 设置单个审计日志文件的大小，可以通过 [`tidb_audit_log_max_lifetime`](#tidb_audit_log_max_lifetime) 设置触发审计日志轮替的时间，也可以通过 [`audit_log_rotate`](#audit_log_rotate) 函数手动触发一次日志轮替。 用户可以同时设置上述两个变量，满足任一条件都会触发日志轮替。
```

File: new-audit-log.md [hunk 9]
```diff
@@ -0,0 +1,537 @@
...
+
+## 日志文件保留数量与时间
+
+为了防止审计日志文件占据过多存储空间，TiDB 提供系统变量来控制审计日志保留的数量与时间：
```
Comment (songrijie) [inline new-audit-log.md:214]:
```suggestion
为了实现对审计归档文件的自动管理，TiDB 提供系统变量来控制审计日志保留的数量与时间：
```

File: new-audit-log.md [hunk 10]
```diff
@@ -0,0 +1,537 @@
...
+
+* [`tidb_audit_log_reserved_backups`](#tidb_audit_log_reserved_backups) 控制每台 TiDB 服务器上保留的审计日志文件的数量
```
Comment (songrijie) [inline new-audit-log.md:216]:
```suggestion
* [`tidb_audit_log_reserved_backups`](#tidb_audit_log_reserved_backups) 控制每台 TiDB 服务器上保留的审计日志文件的数量。结合日志文件大小 [`tidb_audit_log_max_size`](#tidb_audit_log_max_size) 的设置，可以限制总体日志大小的上限。
```

File: new-audit-log.md [hunk 11]
```diff
@@ -0,0 +1,537 @@
...
+
+### `tidb_audit_log_max_size`
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 类型：整数型
+- 默认值：`100`，单位 `MB`
+- 范围：`[0, 102400]`，最大值对应 `100GB`
+- 设置 TiDB 审计日志的文件大小上限，超过该上限将触发[日志轮替](#日志轮替)。
+- 设置为 `0` 时会自动设置为 `100`
```
Comment (songrijie) [inline new-audit-log.md:352]: `0` 应该是代表不会轮替，持续记录在一个文件中。

File: new-audit-log.md [hunk 12]
```diff
@@ -0,0 +1,537 @@
...
+TiDB 中通过 [`audit_log_create_rule`](#audit_log_create_rule) 函数创建过滤器，创建成功后可以通过 [`mysql.audit_log_filter_rules`](#mysqlaudit_log_filter_rules) 查询过滤器。
+
+过滤规则创建成功后，过滤器将对所有指定的审计用户生效。可以通过 [`audit_log_enable_rule`](#audit_log_enable_rule) 启用一个过滤规则，通过 [`audit_log_disable_rule`](#audit_log_disable_rule) 禁用一个过滤规则。
+
+> **注意：**
+>
+> 对于一条审计日志记录，只要其满足任意一条过滤规则，则会被记录到审计日志中。这意味着如果同一时刻有多条对于同一记录冲突的过滤规则生效，则该记录仍会被记录到日志中。
+> 
+> 例如，有一个名为 `visit_test` 的过滤器 `{"filter":[{"table":["test.*"]}]}` 用于过滤所有访问 `test` 数据库的操作，另一个名为 `not_visit_test` 的过滤器 `{"filter":[{"table_excl":["test.*"]}]}` 用于排除所有访问 `test` 数据库的操作。当这两个过滤器对应的过滤规则同时生效时，所有对 `test` 数据库的操作因为满足 `visit_test` 而将被审计日志记录。
+
```
Comment (songrijie) [inline new-audit-log.md:191]:
```suggestion

只要审计功能打开， `AUDIT`类型的所有事件将会被强制审计，不会被任何规则移除。 

```

File: new-audit-log.md [hunk 13]
```diff
@@ -0,0 +1,537 @@
...
+TiDB 审计日志有以下事件类型：
+
+| **事件类型** | **描述** | **父类型** |
+|---|---|---|
+| `CONNECTION` | 记录所有与连接相关的操作，包括握手、建立连接、断开连接、重设连接、变更用户等 | - |
+| `CONNECT` | 记录连接过程中的握手操作 | `CONNECTION` | 
+| `DISCONNECT` | 记录断开连接的操作 | `CONNECTION` |
+| `CHANGE_USER` | 记录变更用户的操作 | `CONNECTION` |
+| `QUERY` | 记录所有执行 SQL 语句的操作 | - |
```
Comment (songrijie) [inline new-audit-log.md:36]:
```suggestion
| `QUERY` | 记录所有执行 SQL 语句的操作， 包括所有对数据的查询和修改报错。  | - |
```

File: new-audit-log.md (hunk 11 above)
Comment (songrijie) [reply new-audit-log.md:355]:
```suggestion
- 在当前版本中，设置为 `0` 会自动设置为 `100`，单日志最大为 `102400` MB 。 后续版本 `0` 将会代表不做日志轮替。 
```

File: new-audit-log.md [hunk 14]
```diff
@@ -0,0 +1,540 @@
...
+---
+
+# TiDB 审计日志
+
+数据库审计是 TiDB 企业版的重要功能，为企业提供了一个强大的监管和审计工具。审计可以记录所有的查询语句、事务操作、用户登录等信息，并且可以将这些信息存储到审计日志中，以备将来的查询和分析，从而保证企业的数据安全和合规性。数据库审计功能可以帮助企业管理人员追踪数据库操作的来源和影响，确保数据不被非法窃取或篡改。同时，数据库审计还可以帮助企业遵守各种法规和合规要求，保障企业在法律和道德方面的合规性。
+
+> **注意：**
+>
+> 数据库审计是 [TiDB 企业版](https://cn.pingcap.com/product/#SelectProduct) 提供的功能，请[咨询 TiDB 顾问团队](https://cn.pingcap.com/product/#SelectProduct)以获取 TiDB 企业版，同时享受商业专家的支持服务。
+
```
Comment (songrijie) [inline new-audit-log.md:14]:
```suggestion

## TiDB 审计日志的覆盖范围

所有用户接口的操作都在审计范围内， 包括 prepare statement 接口。 由此类操作派生(recursive)出的数据库内部操作不在审计范围内。 

**下列外部工具操作在审计范围内：**
- TiCDC 的写入
- 通过 DM 的写入
- 利用 SQL 接口的 BR 命令发起的备份操作

**下列操作不在审计范围内：**
- 数据库系统内部操作
- 由用户操作派生出的数据库操作
- 通过 lightening 的导入操作
- TiSpark 相关操作
- 利用外部 BR 工具发起的备份操作
- 数据库启动、停止、扩展和收缩操作

```

**Pull request closed without merging**